meerevm/cmd/accountcmd.go

```diff
@@ -288,6 +293,33 @@ func accountImport(ctx *cli.Context) error {
 	if err != nil {
 		utils.Fatalf("Could not create the account: %v", err)
 	}
-	fmt.Printf("Address: {%x}\n", acct.Address)
+	fmt.Printf("PKAAddress: {%x}\n", acct.Address)
+	addrs, err := GetQngAddrsFromPrivateKey(hex.EncodeToString(key.D.Bytes()), params.ActiveNetParams.Params)
+	if err != nil {
+		utils.Fatalf("Could not create the account: %v", err)
+	}
+	for _, addr := range addrs {
+		if _, ok := addr.(*address.PubKeyHashAddress); ok {
+			fmt.Printf("PKHAddress: {%s}\n", addr.String())
+		} else {
+			fmt.Printf("PKAddress: {%s}\n", addr.String())
+		}
+	}
 	return nil
 }
+func GetQngAddrsFromPrivateKey(privateKeyStr string, param *params.Params) ([]types.Address, error) {
```

Comment:
1. need not to get addr from private key anyway. (pub key is enough)
2. should use the utility api to handle addr convert/encoding etc, instead of manually handle it here.